teuthology/orchestra/remote.py

```diff
@@ -401,6 +401,13 @@ def get_file(self, path, sudo=False, dest_dir='/tmp'):
             self.remove(path)
         return local_path
 
+    def write_file(self, path, data, sudo=False):
+        """
+        Write a file on a remote machine.
+        """
+        args = 'sudo tee' if sudo else 'tee'
+        self.run('%s %s' % (args, path), stdin=data)
```

Comment:
This code will fail since args should be provided to run as 'args' parameter.
Here is an example of correct usage:
```
>>> from teuthology.orchestra.remote import Remote as r
>>> from StringIO import StringIO as s
>>> localhost=r('localhost')
>>> localhost.run(args="cat", stdin=s("Hello World!"))
2019-07-08 15:51:40,969.969 INFO:teuthology.orchestra.remote:Trying to reconnect to host
2019-07-08 15:51:41,159.159 INFO:teuthology.orchestra.run.localhost:Running:
2019-07-08 15:51:41,159.159 INFO:teuthology.orchestra.run.localhost:> true
2019-07-08 15:51:41,311.311 INFO:teuthology.orchestra.run.localhost:Running:
2019-07-08 15:51:41,311.311 INFO:teuthology.orchestra.run.localhost:> cat
2019-07-08 15:51:41,364.364 INFO:teuthology.orchestra.run.localhost.stdout:Hello World!
RemoteProcess(client=<paramiko.client.SSHClient object at 0x7fa841bffe50>, args='cat', check_status=True, hostname='localhost')
>>> 
```